.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Automated product release
+concurrency: unity_run
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ closed ]
+
+
+jobs:
+  check-release:
+    name: Check release required
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.merged && endsWith(github.repository, '-private') != true }}
+    outputs:
+      release: ${{ steps.check.outputs.ready }}
+    steps:
+      - name: Checkout actions
+        uses: actions/checkout@v2
+        with:
+          repository: pubnub/client-engineering-deployment-tools
+          ref: v1
+          token: ${{ secrets.GH_TOKEN }}
+          path: .github/.release/actions
+      - id: check
+        name: Check pre-release completed
+        uses: ./.github/.release/actions/actions/checks/release
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+  publish:
+    name: Publish package
+    runs-on: ubuntu-latest
```

Comment:
UnityEditor is not available for Linux, this should be macOS

Reply:
Yep, missed this.